JustTweak/Classes/Core/TweaksConfigurationsCoordinator.swift

```diff
@@ -128,4 +128,7 @@ import Foundation
         tweaksCache = [String : TweakCachedValue]()
     }
     
+    @objc public func isTweaksCacheEmpty() -> Bool {
```

Comment:
If this method is to ease the unit tests, it should be internal. Mind that #5 also removed @objc and @objcMembers.

Reply:
I am going to put all unit test in other pr - to a JustEat Tests. But I need this method  to implement Unit test in JustEat Tests.